docs/README.md

```diff
@@ -14,27 +12,7 @@ The following JSON is the default configuration for the module and can be overri
 		"altis": {
 			"modules": {
 				"seo": {
```

Comment:
Don't we intend to remove the whole `modules.seo` block?

Reply:
There is one part of the SEO module remaining: redirections. So it is ok to still have this block.